app/controllers/tweets_controller.rb

```diff
@@ -0,0 +1,77 @@
+class TweetsController < ApplicationController
+
+  get '/tweets' do
+   if logged_in?
+     @tweets = Tweet.all
+     erb :'/tweets/tweets'
+   else
+     redirect to '/login'
+   end
+ end
+
+ get '/tweets/new' do
+   if !logged_in?
+     redirect to '/login'
+   else
+     erb :'/tweets/create_tweet'
+   end
+  end
+
+
+  post '/tweets' do
+    if params[:content] == ""
```

Comment:
I think you could use [ActiveRecord validations](https://learn.co/tracks/full-stack-web-development/sinatra/activerecord/user-authentication-in-sinatra) (e.g. `validates_presence_of :content`) to perform these types of checks more easily.